lib/src/layer/polygon_layer/painter.dart

```diff
@@ -238,11 +276,107 @@ class _PolygonPainter<R extends Object> extends CustomPainter
       return WorldWorkControl.visible;
     }
 
-    // Main loop constructing batched fill and border paths from given polygons.
+    Path holePaths = Path();
+
+    void addPolygon(List<Offset> offsets) {
+      // ignore: dead_code
+      if (!_fillInvertedHoles) return;
```

Comment:
Is ignoring this lint really required? If so, please add a comment why.

Reply:
This is retained for debugging purposes. Added comment.